Gemfile.lock

```diff
@@ -151,6 +151,8 @@ GEM
       rake
     figaro (1.1.1)
       thor (~> 0.14)
+    font-awesome-sass (4.7.0)
```

Comment:
We do not use font-awesome right now. We do not want to blow Zammad right now with additional forms.